lib/SPIRV/SPIRVReader.cpp

```diff
@@ -3100,15 +3120,38 @@ void generateIntelFPGAAnnotationForStructMember(
     Out << E->getMemberDecorationStringLiteral(DecorationUserSemantic,
                                                MemberNumber)
                .front();
+
+  unsigned LSUParamsBitmask = 0;
+  llvm::SmallString<32> AdditionalParamsStr;
+  llvm::raw_svector_ostream ParamsOut(AdditionalParamsStr);
+  if (E->hasMemberDecorate(DecorationBurstCoalesceINTEL, 0, MemberNumber))
+    LSUParamsBitmask |= 0x1;
```

Comment:
Sorry for the late idea: may we add enum for these bitmask values?

Reply:
It's actually a valuable idea, and I've also been thinking of doing this at earlier stages. I'll see if I can create some `enum` that can be used both by the Writer and the Reader.